tests/unit/predicates/has-class-test.js

```diff
@@ -87,6 +87,36 @@ test('resets scope', function(assert) {
   assert.ok(page.foo);
 });
 
+
+test('throws error if selector matches more than one element', function(assert) {
+  fixture(`
+    <span class="lorem"></span>
+    <span class="ipsum"></span>
+  `);
+
+  let page = create({
+    foo: hasClass('lorem', 'span')
+  });
+
+  assert.throws(
+    () => page.foo,
+    /span matched more than one element. If this is not an error use { multiple: true }/
+  );
+});
+
+test('matches multiple elements with multiple: true option', function(assert) {
+  fixture(`
+    <span class="lorem"></span>
+    <span class="ipsum"></span>
+  `);
+
+  let page = create({
+    foo: hasClass('lorem', 'span', { multiple: true })
+  });
+
+  assert.ok(page.foo);
+});
```

Comment:
We need a test for the case when one element has the class and another element doesn't. It should return false in that case.